code/go/0chain.net/smartcontract/faucetsc/sc.go

```diff
@@ -57,6 +58,24 @@ func (fc *FaucetSmartContract) GetRestPoints() map[string]smartcontractinterface
 	return fc.SmartContract.RestHandlers
 }
 
+func (fc *FaucetSmartContract) GetCost(t *transaction.Transaction, funcName string, balances c_state.StateContextI) int {
```

Comment:
We should return error if any, the caller needs to know what happened and take action base on the returned value and error.